<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uploadWindow</class>
 <widget class="QDialog" name="uploadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>업로드</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>이곳을 클릭하세요.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="uploadButton">
     <property name="text">
      <string>업로드</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
